common/beerocks/bwl/CMakeLists.txt

```diff
@@ -54,6 +54,8 @@ if(BWL_TYPE STREQUAL "DWPAL")
     )
 
     find_package(dwpal REQUIRED)
+    find_package(nl-3 REQUIRED)
```

Comment:
libnl3 should be removed from include_directories, above. The target_link_libraries should take care of it (except it doesn't because Findnl-3.cmake is wrong, see my other comment).